### ISSUE TITLE
1.8.8 PDF.Slicer.exe Virustotal False Positive ?

### ISSUE THREAD

**Issue by westhighr** [junrrein/pdfslicer#232]
Hi, great product, thanks for doing the work to develop something like this. Unfortunately, when I scanned the Windows .exe file, two threats were found. Is this a false positive?
https://www.virustotal.com/gui/file/34b11630e3b92ceebf9551c84b46c5258b1cf0038ed2be087e070be133c42e74/detection

**Comment by ziadOUA** [comment]
Yes, when you only see few positives, it is generally a false positive @westhighr
